tests/api.c

```diff
@@ -50593,6 +50592,74 @@ static int test_wolfSSL_SMIME_write_PKCS7(void)
 #endif /* HAVE_SMIME */
 #endif /* !NO_BIO */
 
+#if defined(OPENSSL_ALL) && !defined(NO_FILESYSTEM) && \
```

Comment:
Please move this long macro expression into the function and remove it from the calling function below. So always call this function and just gate the internal function content. Then we only have one copy of this.

Reply:
Good point!  Fixed